DXRMapFixups/DeusEx/Classes/DXRFixupVandenberg.uc

```diff
@@ -704,19 +728,46 @@ function TimerMapFixes()
         break;
 
     case "14_Oceanlab_silo":
-        prevMapsDone = dxr.flagbase.GetBool('Heliosborn') &&  //Finished Vandenberg, mission 12
-            dxr.flagbase.GetBool('schematic_downloaded'); //Finished Ocean Lab, mission 14,
-        prevMapsDone = prevMapsDone || !#defined(injections) || dxr.flags.settings.goals<=0;
+        // by design, no infolink plays after killing Howard Strong if the missile hasn't been redirected
 
-        if(prevMapsDone
-           && dxr.flagbase.GetBool('missile_launched') //Redirected the missile in Silo, mission 14
-           && !dxr.flagbase.GetBool('MS_HowardStrongUnhidden'))
-        {
-            foreach AllActors(class'#var(prefix)HowardStrong', hs) {
-                hs.EnterWorld();
+        if (!dxr.flagbase.GetBool('DXR_SiloEscapeHelicopterUnhidden') && dxr.flagbase.GetBool('missile_launched')) {
```

Comment:
this is getting to be a lot of indentation, maybe could be split into 2 functions